libs/portals/admin/ids-admin/src/components/Application/Application.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Tag,
+  Text,
+  Stack,
+  Box,
+  Select,
+} from '@island.is/island-ui/core'
+import React, { useState } from 'react'
+import BasicInfo from './BasicInfo'
+import Translations from './Translations'
+import ApplicationsUrl from './ApplicationsUrl'
+import Lifetime from './Lifetime'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import { useLoaderData } from 'react-router-dom'
+import {
+  AuthApplicationApplicationUrlList,
+  AuthApplicationLBasicInfoList,
+  AuthApplicationList,
+  AuthApplicationTranslationList,
+  AuthApplicationLifeTimeList,
+} from './Application.loader'
+
+const Application = () => {
+  const applications = useLoaderData() as AuthApplicationList
+  const [application] = useState<AuthApplicationList[0]>(
+    applications[0] ? applications[0] : ({} as AuthApplicationList[0]),
+  )
+  const { formatMessage } = useLocale()
+  const [selectedEnvironment, setSelectedEnvironment] = useState<string>(
+    application.defaultEnvironment.environment,
+  )
+
+  const getEnv = () => {
+    if (!selectedEnvironment) return application.defaultEnvironment
+    return application.environments.find(
+      (env) => env.environment === selectedEnvironment,
+    )
+  }
```

Comment:
Don´t need a function for this, just create a variable instead, like `const selectedEnvironmentData = application.environments.find(... selectedEnvironment)`

Reply:
Solved in https://github.com/island-is/island.is/pull/10534